metadata-ingestion/README.md

```diff
@@ -478,7 +478,11 @@ source:
 Extracts:
 
 - List of databases
-- List of collections in each database
+- List of collections in each database and infers schemas for each collection
+
+By default, schema inference samples 1,000 documents from each collection. Setting `schemaSamplingSize: 0` will scan the entire collection.
```

Comment:
Seems like `schemaSamplingSize: null` will do that, but setting it to 0 will fail config validation because of `PositiveInt`

Reply:
@kevinhu : could you fix this?